src/card/__tests__/card-image-object.scenario.js

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright (c) 2018-2020 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import * as React from 'react';
+
+import {Card, type ImagePropsT, StyledBody} from '../index.js';
+import {styled} from '../../styles/index.js';
+
+export const name = 'card-image-object';
+
+const Container = styled('div', {width: '328px'});
+
+const headerImage: ImagePropsT = {
+  src: 'https://source.unsplash.com/user/erondu',
+  srcSet:
+    'https://source.unsplash.com/user/erondu/500x200, https://source.unsplash.com/user/erondu/600x300 1.5x',
```

Comment:
@nbdn we have to change this to a fix image - the VRT will keep changing with every run

Reply:
Allright, it's now done, to avoid adding a new base64, I reused the thumbnail's base64